Sources/OpenAI/Public/Service/DefaultOpeanAIService.swift

```diff
@@ -14,14 +14,14 @@ struct DefaultOpenAIService: OpenAIService {
 
    private let sessionID = UUID().uuidString
    /// [authentication](https://platform.openai.com/docs/api-reference/authentication)
-   private let apiKey: String
+   private let apiKey: Authorization
    /// [organization](https://platform.openai.com/docs/api-reference/organization-optional)
    private let organizationID: String?
    
    private static let assistantsBeta = "assistants=v1"
       
    init(
-      apiKey: String,
+      apiKey: Authorization,
```

Comment:
We could add an `assert`, so `DefaultOpenAIService` can only instantiated with a `.bearer` Authorization

Reply:
Thanks! wondering if you could use this instead https://github.com/jamesrochabrun/SwiftOpenAI?tab=readme-ov-file#azure-openai-1
`DefaultOpenAIAzureService` should work with Azure although now just supports Completions API. 
Let me know if that is what you need!

Reply:
I changed it back so the default OpenAI API will always use bearer authorization.